packages/core/src/components/globals/globals.tsx

```diff
@@ -151,3 +151,31 @@ export const Flex: React.FC<FlexBoxProps> = ({
 		children,
 	);
 };
+
+export type SectionProps = {
+	ariaLabel: string;
+	testId?: string;
+	className?: string;
+	cfg?: SpaceProps;
+};
+export const Section: React.FC<SectionProps> = ({
+	ariaLabel,
+	testId,
+	className,
+	children,
+	cfg = {},
+	...props
+}) => {
+	const classNames = useClassNames(cfg, [className]);
+	return createElement(
+		'section',
+		{
+			role: 'region',
```

Comment:
@allanthompson-tpr This got merged before I could comment again. [Adding `role="region"` does nothing here](https://w3c.github.io/using-aria/#aria-does-nothing), because `<section>` has that role anyway if it has an accessible name (ie the `aria-label`). I don't think it's actually harmful though, aside from making the JS download slightly bigger.

Reply:
@sussexrick Please see https://github.com/thepensionsregulator/react-components/pull/351 where I've removed the explicit setting of the role.